content/en/blog/_posts/2020-08-25-kubernetes-release-1.19.md

```diff
@@ -0,0 +1,117 @@
+
+## Other Updates 
+### Graduated to Stable
+* [Seccomp](https://github.com/kubernetes/enhancements/issues/135)
+* [Kubelet Client TLS Certificate Rotation](https://github.com/kubernetes/enhancements/issues/266)
+* [Limit node access to API](https://github.com/kubernetes/enhancements/issues/279)
+* [Redesign Event API](https://github.com/kubernetes/enhancements/issues/383)
+* [Graduate Ingress to V1](https://github.com/kubernetes/enhancements/issues/1453)
+* [CertificateSigningRequest API](https://github.com/kubernetes/enhancements/issues/1513)
+* [Building Kubelet without Docker](https://github.com/kubernetes/enhancements/issues/1547)
+
+### Major Changes
+* [Node Topology Manager](https://github.com/kubernetes/enhancements/issues/693)
+* [New Endpoint API](https://github.com/kubernetes/enhancements/issues/752)
+* [Increase Kubernetes support window to one year](https://github.com/kubernetes/enhancements/issues/1498)
+
+### Other Notable Features
+* [Run multiple Scheduling Profiles](https://github.com/kubernetes/enhancements/issues/1451)
+* [CertificateSigningRequest API](https://github.com/kubernetes/enhancements/issues/1513)
+* [Immutable Secrets and ConfigMaps](https://github.com/kubernetes/enhancements/issues/1412)
+
+## Release Notes
+Check out the full details of the Kubernetes 1.19 release in our release notes.
+
+## Availability
+
+Kubernetes 1.19 is available for download on [GitHub](https://github.com/kubernetes/kubernetes/releases/tag/v1.19.0). To get started with Kubernetes, check out these [interactive tutorials](https://kubernetes.io/docs/tutorials/) or run local Kubernetes clusters using Docker container “nodes” with [KinD](https://kind.sigs.k8s.io/) (Kubernetes in Docker). You can also easily install 1.19 using [kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/). 
+
+## Release Team
+This release is made possible through the efforts of hundreds of individuals who contributed both technical and non-technical content. Special thanks to the [release team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.19/release_team.md) led by Taylor Dolezal, Senior Developer Advocate at HashiCorp. The 34 release team members coordinated many aspects of the release, from documentation to testing, validation, and feature completeness. 
+
+As the Kubernetes community has grown, our release process represents an amazing demonstration of collaboration in open source software development. Kubernetes continues to gain new users at a rapid pace. This growth creates a positive feedback cycle where more contributors commit code creating a more vibrant ecosystem. Kubernetes has had over [49,000 individual contributors](https://k8s.devstats.cncf.io/d/24/overall-project-statistics?orgId=1) to date and an active community of more than 3,000 people.
+
```

Comment:
```suggestion

## Release Logo

All of you inspired this Kubernetes 1.19 release logo! This release was a bit more of a marathon and a testament to when the world is a wild place, we can come together and do unbelievable things. 

"Accentuate the Paw-sitive" was chosen as the release theme because it captures the positive outlook that the release team had, despite the state of the world. The characters pictured in the 1.19 logo represent everyone's personalities on our release team, from emo to peppy, and beyond! 

About the designer: Hannabeth Lagerlof is a Visual Designer based in Los Angeles, California, and she has an extensive background in Environments and Graphic Design. Hannabeth creates art and user experiences that inspire connection. You can find Hannabeth on Twitter as @emanate_design.
```

Reply:
I will send the image to you, @mkorbi, later today!

Reply:
can't wait for it!